board/board.py

```diff
@@ -0,0 +1,66 @@
+# CLI 기반 게시판 프로그램을 개발하세요.
+# 모든 '게시글'은 '제목'과 '내용'으로 이루어집니다.
+# '글 추가'와 '글 목록', '글 삭제' 기능을 구현하면 됩니다.
+from dataclasses import dataclass, field
+
+
+@dataclass
+class Post:
+    title: field()
+    content: field()
+
+
+class Board:
+
+    def __init__(self):
+        self.posts = []
+
+    def post(self, title, content):
+
+        post = Post(title, content)
+        self.posts.append(post)
```

Comment:
```suggestion
        self.posts.append(Post(title, content))
```

굳이 변수가 따로 있을 필요가 없겠다. 이걸 리턴하거나 그러고 있질 않으니깐